openapi/code_samples/PHP-SDK-3.0/authentication-options/put.php

```diff
@@ -0,0 +1,11 @@
+$service = new Rebilly\Sdk\Service($client);
+
+$authenticationOptionsForm = Rebilly\Sdk\Model\AuthenticationOptions::from([]);
```

Comment:
`from` method is only useful with non-empty argument or for chained calls.
```suggestion
$authenticationOptionsForm = new Rebilly\Sdk\Model\AuthenticationOptions();
```